src/main/java/com/github/nylle/javafixture/Reflector.java

```diff
@@ -64,6 +69,26 @@ private Stream<Field> getDeclaredFields(Class<?> type) {
                         .orElse(Stream.of()));
     }
 
+    public Annotation[] getFieldAnnotations(Field field) {
+        try {
+            return getFieldAnnotations(field, clazz).toArray(Annotation[]::new);
+        } catch (IntrospectionException e) {
+            return field.getAnnotations();
+        }
+    }
+
+    public Stream<Annotation> getFieldAnnotations(Field field, Class<?> type) throws IntrospectionException {
```

Comment:
This method doesn't need to be public. If you make it private, no test is necessary.